features/in_onderzoek.feature

```diff
@@ -356,83 +450,75 @@ Functionaliteit: in onderzoek
 
     Scenario: persoon heeft hele groep naam in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                        | waarde    |
-      | burgerservicenummer         | 555550001 |
-      En de persoon heeft de volgende 'persoonInOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | 010200   |
-      | datumIngangOnderzoek          | 20220307 |
+      | naam                                        | waarde    |
+      | burgerservicenummer                         | 555550001 |
+      | aanduiding gegevens in onderzoek (01.83.10) | 010200    |
+      | datum ingang onderzoek (01.83.20)           | 20220307  |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde        |
+      | geslachtsnaam (02.40) | geslachtsnaam |
```

Comment:
De waarde 'geslachtsnaam' is geen reële naar, wijzig dit in bijv. 'Groenen'. Deze wordt overigens verder niet gebruikt in dit scenario.